inventory/byo/hosts.example

```diff
@@ -21,6 +21,9 @@ ansible_ssh_user=root
 # deployment type valid values are origin, online and enterprise
 deployment_type=atomic-enterprise
 
+# Install the openshift examples
+#install_openshift_examples=true
```

Comment:
This should probably be named openshift_install_examples.

I would also rename the fact to openshift.common.install_examples